src/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { FilterBy } from './types/FiilterBy';
+import cn from 'classnames';
+
+interface Props {
+  onFilterClick: (value: FilterBy) => void;
+  activeTodosCount: number;
+  onClearCompleted: () => void;
+  selectedFilterBy: FilterBy;
+}
+
+export const Footer: React.FC<Props> = ({
+  onFilterClick,
+  activeTodosCount,
+  onClearCompleted,
+  selectedFilterBy,
+}) => (
+  <footer className="todoapp__footer" data-cy="Footer">
+    <span className="todo-count" data-cy="TodosCounter">
+      {activeTodosCount} items left
+    </span>
+
+    <nav className="filter" data-cy="Filter">
+      <a
```

Comment:
You can show these elements using `.map()`

Reply:
> You can show these elements using `.map()`

Good point